### PR TITLE
fixes dsi dump godmode9i

### PR DESCRIPTION
**Pull request by quadsYT**
The old link doesnt work anymore, so im updating it to fix it
Couldnt test, have bad internet so cant download docker + it just changes the link

### CODE REVIEW

File: cogs/assistance.py [hunk 1]
```diff
@@ -1049,7 +1049,7 @@ async def dump(self, ctx, console=None):
             await ctx.send(embed=embed)
         elif self.check_console(console, channel_name, 'dsi'):
             embed = discord.Embed(title="GodMode9i dump Guide", color=ConsoleColor.legacy())
-            embed.set_author(name="NightScript", url="https://dsi.cfw.guide/dumping-cartridges")
+            embed.set_author(name="NightScript", url="https://dsi.cfw.guide/dumping-game-cards")
             embed.url = "https://dsi.cfw.guide/dumping-cartridges"
```
Comment (lifehackerhansol) [inline cogs/assistance.py:1053]: This still leads to the old link.

**Review by lifehackerhansol — CHANGES_REQUESTED** [review]
Works, partially. See review

File: cogs/assistance.py (hunk 1 above)
Comment (quadsYT) [reply cogs/assistance.py]: fixed it

**Review by lifehackerhansol — APPROVED** [review]
Works.

**Pull request merged**